src/components/Text/Text.js

```diff
@@ -16,8 +16,7 @@ const Text = styled.p`
   line-height: 1.4;
   margin: 0;
   margin-bottom: ${spacingSmall};
-  color: ${({ highlight }) => (highlight ? colorTurquoiseDimmed : colorBlack)};
-  color: ${({ dimmed }) => (dimmed ? colorDarkGrey : colorBlack)};
+  color: ${({ highlight, dimmed }) => (highlight ? colorTurquoiseDimmed : dimmed ? colorDarkGrey : colorBlack)};
```

Comment:
What do you think about 
```
function getFontColor({ highlight, dimmed }) {
  if (highlight) {
    return colorTurquoiseDimmed;
  } else if (dimmed) {
    return colorDarkGrey;
  }

  return colorBlack;
}

const Text = styled.p`
{...}
color: ${getFontColor};
`
```
Complex if-else shorthands can be confusing.

Reply:
I agree ;)